src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -101,6 +101,7 @@
     <Compile Include="Utf8JsonReaderTests.TryGet.Date.cs" />
     <Compile Include="Utf8JsonReaderTests.ValueTextEquals.cs" />
     <Compile Include="Utf8JsonWriterTests.cs" />
+    <Compile Include="JsonNodeTests.JsonNumber.cs" />
```

Comment:
Based on previous discussion, let's move this to its own item group for now (both in source and test csproj). We can merge them back after 3.0.